docs/environment-variables.md

```diff
@@ -30,6 +30,10 @@ If you need a `.env_example` file as a template, use the `.env_example` file in
 | RPC_USER                           | no       | multichainrpc                     | api/bc               | The user used to connect to the multichain daemon.                                                                                                                                                                                                                                                                                                                                              |
 | EXTERNAL_IP                        | no       |                                   | bc                   | The IP address with which the current node can be reached. Example: If you have a VM running on 52.52.52.52 and you want to start a slave node from this VM to connect to a master running on 53.53.53.53, you set the `EXTERNAL_IP` to 52.52.52.52 on this node.                                                                                                                               |
+| NODE_ENV                           | yes      | development or production         | bc                   | Environment: Default development when running development-script. Production when running production-script                                                                                                                                                                                                                                                                                     |
+| CERT_PATH                          | no       |                                   | bc                   | The path to the certificate used by the blockchain to authenticate with the connection peer. Note that self-signed certificates are not allowed in production environments.                                                                                                                                                                                                                     |
+| CERT_CA_PATH                       | no       |                                   | bc                   | The path to the certificate authority root certificate by the blockchain to authenticate with the connection peer. Note that self-signed certificates are not allowed in production environments.                                                                                                                                                                                               |
+| CERT_KEY_PATH                      | no       |                                   | bc                   | The path to the certificate key used by the blockchain to authenticate with the connection peer.                                                                                                                                                                                                                                                                                                |
```

Comment:
add these env var so blockchain/README.md

also describe the new feature in the README and maybe link to a wikipedia entry that explains this certificate pattern